packaging/fedora/snapd.spec

```diff
@@ -66,7 +66,8 @@
 # buildmode PIE triggers external linker consumes -extldflags
 %define gobuild_static(o:) go build -buildmode pie -compiler gc -tags=rpm_crashtraceback -ldflags "${LDFLAGS:-} -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '%__global_ldflags -static'" -a -v -x %{?**};
 %endif
-%if 0%{?rhel} == 7
+# These macros are not defined in RHEL 7 & 8
+%if 0%{?rhel} >= 7
```

Comment:
The RHEL 7 macro should be left alone. Instead, we should change the `%if 0%{?fedora}` conditional for `%gobuild_static` to `%if 0%{?fedora} || 0%{?rhel} >= 8`.

Reply:
Ok, will do